app/models/stripe_wrapper.rb

```diff
@@ -0,0 +1,29 @@
+module StripeWrapper
+  class Charge
+    attr_reader :error_message, :response
+    def initialize(options={})
+      @response = options[:response]
+      @error_message = options[:error_message]
+    end
+
+    def self.create(options={})
+      begin
+        response = Stripe::Charge.create(
+          amount: options[:amount],
+          currency: 'usd',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.